fluent-package/msi/assets/fluentd.bat

```diff
@@ -12,6 +12,9 @@ set FLUENT_PACKAGE_TOPDIR=%FLUENT_PACKAGE_TOPDIR:\=/%
 
 set PATH=%FLUENT_PACKAGE_TOPDIR%bin;%PATH%
 set PATH=%FLUENT_PACKAGE_TOPDIR%;%PATH%
+for /f "usebackq" %%i in (`^""%FLUENT_PACKAGE_BINDIR%\ruby.exe" -rrbconfig -e "print RbConfig::CONFIG['ruby_version']"^"`) do set RUBY_VERSION=%%i

```

Comment:
It seems that no %FLUENT_PACKAGE_BINDIR% in fluentd.bat.
(it exists with fluent-gem.bat)

Reply:
Thanks. I added